test/e2e/elasticache/replication_group/coverage_tests/rg_creation.sh

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env bash
```

Comment:
Any reason to place this test in a `coverage_tests` subdirectory? Why not just `test/e2e/elasticache/replication_group/create_fail.sh`?

Reply:
I think the rationale was to differentiate them from our basic e2e tests, since many of the files contain more complex scenarios and negative/failure cases